### PR TITLE
OSDOCS-RHDEVDOCS-4304: shared resource updates for validating admission webhooks

### PR DESCRIPTION
**Pull request by gabemontero**
Version(s): 4.13

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-4304
https://issues.redhat.com/browse/RHDEVDOCS-4338

/assign @rolfedh 
@Preeticp FYI

per prior discussion with @Preeticp I'm tagging @rolfedh to help  get my doc contributions here over the finish line

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod.adoc [hunk 1]
```diff
@@ -10,9 +10,9 @@ To access a `SharedConfigMap` custom resource (CR) instance from a pod, you gran
 
 * You have created a `SharedConfigMap` CR instance for the config map that you want to share across namespaces in the cluster.
 * You must have permission to perform the following actions:
-** Create build configs and start builds.
 ** Discover which `SharedConfigMap` CR instances are available by entering the `oc get sharedconfigmaps` command and getting a non-empty list back.
-** Determine if the `builder` service accounts available to you in your namespace are allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the `builder` service account in your namespace is listed.
+** Determine if the service account your pod will specify is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
```
Comment (rolfedh) [inline modules/ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod.adoc:14]:
```suggestion
** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
```

File: modules/ephemeral-storage-csi-inline-overview.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ This feature allows you to specify CSI volumes directly in the `Pod` specificati
 By default, {product-title} supports CSI inline ephemeral volumes with these limitations:
 
 * Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.
-* The Shared Resource CSI Driver supports inline ephemeral volumes as a Technology Preview feature.
+* The Shared Resource CSI Driver supports inline ephemeral volumes as a Technology Preview feature, but only for the purpose of accessing `Secrets` or `ConfigMaps` across multiple namespaces.
```
Comment (rolfedh) [inline modules/ephemeral-storage-csi-inline-overview.adoc:18]:
```suggestion
* The Shared Resource CSI Driver supports using inline ephemeral volumes to access `Secrets` or `ConfigMaps` across multiple namespaces as a Technology Preview feature.
```

Comment (gabemontero) [reply modules/ephemeral-storage-csi-inline-overview.adoc:18]: @rolfedh can we say `... inline ephemeral volumes only to access ...`

i.e. add `only`

File: modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc [hunk 3]
```diff
@@ -10,9 +10,9 @@ To access a `SharedSecret` custom resource (CR) instance from a pod, you grant a
 
 * You have created a `SharedSecret` CR instance for the secret you want to share across namespaces in the cluster.
 * You must have permission to perform the following actions
-** Create build configs and start builds.
 ** Discover which `SharedSecret` CR instances are available by entering the `oc get sharedsecrets` command and getting a non-empty list back.
-** Determine if the `builder` service accounts available to you in your namespace are allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the `builder` service account in your namespace is listed.
+** Determine if the service account your pod will specify is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
```
Comment (rolfedh) [inline modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc:14]:
```suggestion
** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
```

File: modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc [hunk 4]
```diff
@@ -10,9 +10,9 @@ To access a `SharedSecret` custom resource (CR) instance from a pod, you grant a
...
+** Determine if the service account your pod will specify is allowed to use `csi` volumes, of if you as the requesting user (if you create the pod directly) is allowed to use `csi` volumes.  See "Understanding and managing pod security admission" for details.
```
Comment (rolfedh) [inline modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc:15]: Maybe:
```suggestion
** Determine if the service account your pod specifies is allowed to use `csi` volumes, or if you, as the requesting user who created the pod directly, are allowed to use `csi` volumes.  See "Understanding and managing pod security admission" for details.
```

**Review by rolfedh — APPROVED** [review]
Looks good to me! I've offered some suggestions for minor style issues. Thanks.

File: modules/ephemeral-storage-csi-inline-overview.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/ephemeral-storage-csi-inline-overview.adoc:18]: > @rolfedh can we say `... inline ephemeral volumes only to access ...`
> 
> i.e. add `only`

Yes. that works.

**Comment by gabemontero** [comment]
> Looks good to me! I've offered some suggestions for minor style issues. Thanks.

thanks @rolfedh .... I accepted all the suggestions as is except for https://github.com/openshift/openshift-docs/pull/56497#discussion_r1119292530 .... are you ok with my addition there?

**Comment by gabemontero** [comment]
ok `only` added in that one spot, commits squashed @rolfedh thanks

**Comment by gabemontero** [comment]
now only is there @rolfedh

**Comment by gabemontero** [comment]
/assign @jitendar-singh 

can you sign off on this per your testing a few weeks ago?  thanks

**Pull request merged**